google-cloud-examples/src/main/java/com/google/cloud/examples/storage/buckets/RemoveBucketIamMember.java

```diff
@@ -35,21 +35,34 @@ public static void removeBucketIamMember(String projectId, String bucketName) {
 
     Policy originalPolicy = storage.getIamPolicy(bucketName);
 
-    // See the StorageRoles documentation for other roles:
-    // https://googleapis.dev/java/google-cloud-clients/latest/com/google/cloud/storage/StorageRoles.html
-    Role role = StorageRoles.objectViewer();
+    String role = "roles/storage.objectViewer";
 
-    // See the Identity documentation for other identities:
-    // https://googleapis.dev/java/google-cloud-core/latest/com/google/cloud/Identity.html
-    Identity identity = Identity.group("example@google.com");
+    String member = "group:example@google.com";
+    List<Binding> bindings = new ArrayList(originalPolicy.getBindingsList());
+    for (int index = 0; index < bindings.size(); ++index) {
+      Binding binding = bindings.get(index);
+      if (binding.getRole().equals(role)
+          && binding.getMembers().contains(member)
+          && null == binding.getCondition()) {
+        bindings.set(index, binding.toBuilder().removeMembers(member).build());
+        break;
+      }
+    }
 
     Policy updatedPolicy =
-        storage.setIamPolicy(
-            bucketName, originalPolicy.toBuilder().removeIdentity(role, identity).build());
+        storage.setIamPolicy(bucketName, originalPolicy.toBuilder().setBindings(bindings).build());
 
-    if (updatedPolicy.getBindings().get(role) == null
-        || !updatedPolicy.getBindings().get(role).contains(identity)) {
-      System.out.printf("Removed %s with role %s from %s\n", identity, role, bucketName);
+    boolean bindingExists = false;
+    for (Binding binding : updatedPolicy.getBindingsList()) {
+      if (binding.getRole().equals(role) && null == binding.getCondition()) {
+        bindingExists = true;
+        if (!binding.getMembers().contains(member)) {
+          System.out.printf("Removed %s with role %s from %s\n", member, role, bucketName);
+        }
+      }
+    }
+    if (!bindingExists) {
```

Comment:
do we want to have an else here? Are you expected to remove a role? Should it throw?